docs/tempo/website/configuration/tls.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Configure TLS Communication
+weight: 70
+---
+
+# Configure TLS Communication
+
+Tempo can be configured to communicate between the components using Transport Layer Security, or TLS.
+
+Please note, the ciphers and TLS version here are for example purposes only. We are in no way recommending which ciphers or TLS versions for use in production environments, and are here for example purposes only.
```

Comment:
Is there a markdowny way for us to call this out a little brighter?

Reply:
`**bold**` ?

Reply:
I see the impulse to specify a person with a title, but not sure what to say about it.

Reply:
I think we're good without saying a specific person or talk to anyone. someone implementing this should know that they need to have any TLS work conform with their own network secirty.